### DISCUSSION TITLE
Is it possible to create two minecraft servers in the same vps?

### DISCUSSION THREAD

**Discussion by AlejandroRN191** [GameServerManagers/LinuxGSM#3500] [Q&A] [answered]
I have a vps, and I want to know if it is possible to create two servers, and if possible, how I do with the ip’s. Sorry my English is not god :(.

**Comment by boatplugs** [comment]
Yes it should be possible. Look into dns _srv records and running two instances of the minecraft server .jar configured for different ports.

https://www.namecheap.com/support/knowledgebase/article.aspx/9765/2208/how-can-i-link-my-domain-name-to-a-minecraft-server/

Essentially this but you will need two _srv records, one for each instance of minecraft. 

I.E mc1.yourdomain.com points to the default minecraft 25565 port at your VPS address
and mc2.yourdomain.com points to your second minecraft instance port on say 25575 pointing to the same VPS address

the _srv records should inform the mincraft client which port it should connect to thus allowing multiple servers running on different listen ports to be accessible from the same IP. Hope this helps!

**Comment by Claiyc** [comment, marked as answer]
To be clear, it is not necessary to utilize srv records in order to run multiple gameservers by any means. Just make sure that you use different ports and rename your mcserver -script to something like mcserver-2. 
You also might want to take a look at this: https://docs.linuxgsm.com/features/multiple-game-servers
